vector/src/main/java/im/vector/app/features/spaces/SpaceCreationActivity.kt

```diff
@@ -106,6 +107,9 @@ class SpaceCreationActivity : SimpleFragmentActivity(), CreateSpaceViewModel.Fac
                     setResult(RESULT_OK, Intent().apply {
                         putExtra(RESULT_DATA_CREATED_SPACE_ID, it.spaceId)
                         putExtra(RESULT_DATA_DEFAULT_ROOM_ID, it.defaultRoomId)
+                        if (it.topology == SpaceTopology.JustMe) {
+                            putExtra(RESULT_DATA_CREATED_SPACE_IS_JUST_ME, true)
+                        }
```

Comment:
I prefer to always put the extra value, even when value is false.

Reply:
done